llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.cpp

```diff
@@ -466,6 +467,11 @@ bool getMUBUFIsBufferInv(unsigned Opc) {
   return Info ? Info->IsBufferInv : false;
 }
 
+bool getMUBUFHasTFE(unsigned Opc) {
```

Comment:
Should this be `getMUBUFIsTFE()`?

Reply:
Maybe - TFE stands for "texture fault enable" so HasTFE feels more logical than IsTFE.

Reply:
I left it as HasTFE - unless you feel strongly that it should be IsTFE?

Reply:
Looks very minor, so I guess should be fine either way. The doubt was, HasTFE might be misleading in presence of `getMUBUFHasSoffset()`, etc. which have a different meaning.

Reply:
I decided to go with getMUBUFTfe - since that mirrors the name of the field.